src/items/equipment/glimpses_of_the_endless_horizon.json

```diff
@@ -0,0 +1,71 @@
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<h2>@UUID[Compendium.sfrpg.rules.P7TMxsRPHGxzlHmh]{Artifact}</h2><p>The Glimpses of the Endless Horizon is a holy text of @UUID[Compendium.sfrpg.setting.KZvxBU9KGbqRCKkv]{Weydan}, known in academic circles for years, but is generally treated as a curiosity for vagabonds and adventurers. The drastic changes to interstellar travel wrought by the Drift Crisis, however, have caused interest in the artifact to spike, and it’s now highly sought by powerful forces across the Pact Worlds and beyond. Reports of the religious text describe wildly differing appearances depending on the beholder, causing theologians and scholars to avidly debate whether it’s one artifact or a series of linked artifacts.</p>\n<p>All forms of this magical logbook share embossing depicting Weydan’s symbolic starship upon it, and contain shorthand navigational notes and charts representing an endless variety of galaxies across multiple planes. It has appeared as an old‑fashioned tome, a compact computer drive running on a never-before-seen coding language, a series of bound and trimmed insectile wings written upon in crystallized amber, and other various, fantastic forms. No matter how the Glimpses of the Endless Horizon appears, the bearer can read the port reports, navigational anecdotes, and charts in their native tongue. Every page describes a unique system, and those who turn back to a previously read entry find an entirely different galaxy described instead.</p>\n<p>The Glimpses of the Endless Horizon serves as a navigational aid in starship travel. You can spend 1 hour consulting the logs to reduce the DC of Piloting checks to navigate or astrogate by 5. If you’re a follower of Weydan who’s touching the Glimpses of the Endless Horizon, you can cast @UUID[Compendium.sfrpg.spells.e6iK8NCKZTNkDffh]{pinpoint navigation} once per day.</p>\n<p>Once per day, the book allows you to cast @UUID[Compendium.sfrpg.spells.uH3JWy7ICH8DNS8m]{augury} about navigational questions, such as “Will we find the answers we’re looking for in this system?” You must spend the 1 Resolve Point required for the spell, but the spell is augmented to look far enough into the future to account for a single trip through the Drift. If you’re a follower of Weydan, the chance for successfully receiving a meaningful reply is 100%.</p>\n<p>The book can be destroyed by leaving it aboard a ship that’s navigated into a star, gas giant, black hole, or some equally massive gravitational phenomenon while both the ship and book are under the effects of a @UUID[Compendium.sfrpg.spells.Et8svAkPZ7RNsJJf]{nondetection} spell. Followers of Weydan claim the book will then immediately remanifest in a new form immediately after it’s destroyed, believing it to possess the same multiform nature as Weydan himself.</p>"
```

Comment:
Add a `>` before the artifact link, so it reads `>Artifact` like other references to parent items. Same goes for the others.